src/ol/View.js

```diff
@@ -1195,7 +1394,8 @@ function animationCallback(callback, returnValue) {
  */
 export function createCenterConstraint(options) {
```

Comment:
This is more a question or something for a separate pull request, but I always wondered what we have this function exported for when we don't have an API to apply a constraint we created with this function. Or do we have an API for that and I'm just missing something?

Reply:
No idea TBH. Probably of a remnant of a past design decision.

Reply:
Probably, yes. Thanks for confirming.

Reply:
Actually I lied: this is exported to be used in tests here:
https://github.com/openlayers/openlayers/blob/3da449d77e6fa76fcdceae7bdbdde61d429455bd/test/spec/ol/view.test.js#L2-L5